### DISCUSSION TITLE
Shuttle location and hes spawn

### DISCUSSION THREAD

**Discussion by M1and1B** [space-wizards/space-station-14#26827] [Questions]
1) Is it possible to somehow change the location of the shuttle spawns from the station? As an example, the spawn distance of the Trade Post?
2) Where can I specify the spawn of the shuttles you need on the game map?

**Comment by CyberDuck-Git** [comment]
It’s better to ask in the discord channel They very rarely answer here

**Comment by KashRas2** [comment]
> 1. Is it possible to somehow change the location of the shuttle spawns from the station? As an example, the spawn distance of the Trade Post?
> 2. Where can I specify the spawn of the shuttles you need on the game map?

Haien Era
